vsts_ci/azure-pipelines.yml

```diff
@@ -0,0 +1,18 @@
+jobs:
+  - job: Linux
+    pool:
+      vmImage: ubuntu-16.04
+    steps:
+      - template: linux/continuous-build-linux.yml
+
+  - job: Windows
+    pool:
+      vmImage: win1803
+    steps:
+      - template: win32/continuous-build-win32.yml
+
+  - job: macOS
```

Comment:
Before we support running CI on macOS, we can temporarily comment this section out. Otherwise we will always see errors no matter whether there are errors in the code.

Reply:
updated